src/briefcase/commands/new.py

```diff
@@ -9,11 +9,25 @@
 from briefcase.config import PEP508_NAME_RE
 from briefcase.exceptions import NetworkFailure
 
+from keyword import iskeyword
+
 from .base import BaseCommand, BriefcaseCommandError
 from .create import InvalidTemplateRepository
 
 VALID_BUNDLE_RE = re.compile(r'[a-zA-Z0-9-]+(\.[a-zA-Z0-9-]+)+$')
 
+javareserved = [
```

Comment:
We also want to avoid calling out Java specifically. If you're not writing an Android app, it's going to be weird to be warning you about a language that you're not using. There are also other languages that we need to care about - at the very least Javascript and Objective C - so this should be a more generic "reserved" list, rather than *Java* reserved specifically.